codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -681,7 +682,7 @@ public boolean isPushPromiseSent() {
          */
         private int nextReservationStreamId;
         private int lastStreamKnownByPeer = -1;
-        private boolean pushToAllowed = true;
+        private boolean pushToAllowed;
```

Comment:
This doesn't look correct as `false` is the default

Reply:
The field is assigned in the constructor, so the initialiser is always overwritten.

Reply:
ah ok... Then its fine :)